cmake_modules/parse_hpcc_chart.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#
+# Common utilities used by tools for automating tagging and release
+#
+set -e
+
+VERSIONFILE=Chart.yaml
+if [ ! -f $VERSIONFILE ]; then
+  echo "Expected $VERSIONFILE not found"
+  exit 2
+fi
+
+
+function parse_cmake()
```

Comment:
Should this have a different name?

Reply:
I think in this version of the PR the intention was to override the original function. But it has changed in the later PR.